.editorconfig

```diff
@@ -0,0 +1,10 @@
+# editorconfig.org
+root = true
+
+[*]
```

Comment:
> [*.{php,css,js,sh,md}]

[Trailing whitespace can be significant in markdown](https://daringfireball.net/projects/markdown/syntax#p) (indicating a hard line break)